core/Interfaces.carp

```diff
@@ -1,5 +1,3 @@
-(load-once "Macros.carp")
```

Comment:
Does this not depend on `Macros.carp` anymore?

Reply:
It does, but we don't load other core files transitively but rather depend on their order in `Core.carp`. Or at least we probably shouldn't do that – to make it (arguably) easier to write your own prelude. Makes sense?

Reply:
Totally; it’s just worth noting that just loading `core/Interfaces.carp` won’t work then because it implicitly expects `Macros.carp`, which is something to keep in mind when writing your own prelude.

Reply:
Alternatively we could make sure that each file in Core (transitively) imports everything it needs. That would be good in that you can just import one higher level module (like e.g. "Test.carp") and be sure that it works.